src/Simplic.Flow.Node/Model/StartWithConditionNode.cs

```diff
@@ -1,13 +1,31 @@
-﻿namespace Simplic.Flow.Node
+﻿using Simplic.Flow.Event;
+
+namespace Simplic.Flow.Node
 {
     public class StartWithConditionNode : ConditionNode
     {
-        protected override bool Compare(IFlowRuntimeService runtime, DataPinScope scope)
+        protected override bool Compare(IFlowRuntimeService runtime, FlowEventArgs args, DataPinScope scope)
```

Comment:
im not sure about giving event args to each node as the nodes can connect pins to the event out pin